Bigtable/src/ResumableStream.php

```diff
@@ -116,6 +121,17 @@ public function __construct(
         $this->callOptions['retrySettings'] = [
             'retriesEnabled' => false
         ];
+
+        $this->delayFunction = function (int $backoffCount) {
+            $initialDelayMillis = 100;
+            $initialDelayMultiplier = 1.3;
+            $maxDelayMillis = 60000;
+
+            $delayMultiplier = $initialDelayMultiplier ** $backoffCount;
+            $delayMs = min($initialDelayMillis * $delayMultiplier, $maxDelayMillis);
```

Comment:
please add jitter

Reply:
We don't support jitter in our `RetrySettings`. How do you think the jitter should be implemented?

Looking at [Java's docs](https://cloud.google.com/java/docs/reference/gax/latest/com.google.api.gax.retrying.RetrySettings.Builder#com_google_api_gax_retrying_RetrySettings_Builder_isJittered__), they do this:

```java
actualDelay = rand_between(0, min(maxRetryDelay, exponentialDelay))
```

In our case it would look like this

```php
$delayMs = min($initialDelayMillis * $delayMultiplier, $maxDelayMillis);
$actualDelay = mt_rand(0, $delayMs);
```

Does that look appropriate?

Reply:
a bvit confused, whats exponentialDelay in this case? is it $delayMs?

Reply:
I dont love full jitter, but it'll be consistent with java gax and definitely better than no jitter. So lets go with that. Looks good, but I think you want ** not *

Reply:
sorry I misunderstood what delayMultiplier was. You had it right the first time

Reply:
Done!